tests/api/nodes_test.py

```diff
@@ -35,6 +35,36 @@ def test_list_all_namespaces(client_with_examples: TestClient) -> None:
     ]
 
 
+def test_list_nodes_by_namespace(client_with_examples: TestClient) -> None:
+    """
+    Test ``GET /namespaces/all/``.
```

Comment:
```suggestion
    Test ``GET /namespaces/{namespace}/``.
```
Also these should be moved to a `namespaces_test.py` file since the logic has been moved there.

Reply:
Makes sense, done!